js/scripts/zone_loaders/village_zone_loaders.js

```diff
@@ -693,9 +693,9 @@ const loadWoods = () => {
             Zone.getZones().village.hillDownFromMain
         ));
         GAME.addEntity(new Border(
-            new Vector(ZONE.MAX_PT.x, 0),
-            new Vector(1, ZONE.PIXEL_SIZE.y),
-            Zone.getZones().village.insideCave
```

Comment:
To clean things up I would suggest putting the border entities side by side at the top of the loader. It would be closer to what has been done in other zones and may help a little bit with code maintainability.

Reply:
I realize that I may have put that code here originally.

Reply:
You're right, I should have moved it. Thanks for noticing that.